packages/kit/src/runtime/server/validate-headers.spec.js

```diff
@@ -0,0 +1,53 @@
+import { describe, test, expect, beforeAll, afterAll, beforeEach, vi } from 'vitest';
+
+describe('validateHeaders', () => {
+	/** @type {typeof console.warn} */
+	let console_warn;
+
+	/** @type {(headers: Record<string, string>) => void} */
+	let validateHeaders;
+
+	beforeAll(() => {
+		console_warn = console.warn;
+		// @ts-expect-error
```

Comment:
Same thing here, I think you can add a global declaration to the top and avoid using `globalThis` here.

Reply:
I found success in the other declarations, but I didn't have much luck silencing it here, despite trying a couple of things 🤔.